From e944b47d541f6d2f24e1dd23910414bb6ba9f849 Mon Sep 17 00:00:00 2001
From: Linu Cherian <linu.cherian@cavium.com>
Date: Tue, 11 Apr 2017 20:12:45 +0530
Subject: [PATCH 20/22] arm64: Documentation: Add Cavium ThunderX2 SMMUv3
 erratas.

Patch-mainline: Submitted, https://patchwork.kernel.org/patch/9675597/
References: bsc#1035000

Add Cavium ThunderX2 SMMUv3 erratas to the errata list.

Signed-off-by: Linu Cherian <linu.cherian@cavium.com>
Message-Id: <1491921765-29475-8-git-send-email-linucherian@gmail.com>
Signed-off-by: Robert Richter <rrichter@cavium.com>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 Documentation/arm64/silicon-errata.txt |    2 ++
 1 file changed, 2 insertions(+)

--- a/Documentation/arm64/silicon-errata.txt
+++ b/Documentation/arm64/silicon-errata.txt
@@ -60,6 +60,8 @@ stable kernels.
 | Cavium         | ThunderX ITS    | #22375, #24313  | CAVIUM_ERRATUM_22375    |
 | Cavium         | ThunderX GICv3  | #23154          | CAVIUM_ERRATUM_23154    |
 | Cavium         | ThunderX Core   | #26026          | CAVIUM_ERRATUM_26026    |
+| Cavium         | ThunderX2 SMMUv3| #74             | N/A                     |
+| Cavium         | ThunderX2 SMMUv3| #126            | N/A                     |
 |                |                 |                 |                         |
 | Freescale/NXP  | LS2080A/LS1043A | A-008585        | FSL_ERRATUM_A008585     |
 | Qualcomm Tech. | Falkor v1       | E1003           | QCOM_FALKOR_ERRATUM_1003|
